libs/cua-driver-fixtures/README.md

```
@@ -0,0 +1,116 @@
+
+```bash
+open libs/cua-driver-fixtures/test_page.html
+```
+
+## Where each port references the canonical copy
+
+Existing port paths are preserved as symlinks. Tests that read these
+paths (`os.path.join(_THIS_DIR, "fixtures", "interactive.html")`,
+`f"{html_server}/test_page.html"`) work unchanged.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)